methods/ga4gh_0.6.0/hhvariant/conf/hhvariant.routes

```diff
@@ -0,0 +1,18 @@
+GET  /variant                          controllers.variant.VariantController.index()
```

Comment:
what would be great is that the path would also contain the version... but dunno yet how this could be with routes